filebeat/prospector/redis/prospector.go

```diff
@@ -49,6 +50,13 @@ func NewProspector(cfg *common.Config, outletFactory channel.OutleterFactory) (*
 	return p, nil
 }
 
+func init() {
```

Comment:
Could we always put the init functions on the top after the imports? I kind of like them at the top as it becomes also obvious they are executed first.